pkg/arvo/app/btc-node-store.hoon

```diff
@@ -0,0 +1,146 @@
+::  btc-node-store: data store for state received from a bitcoin full node
+::
+/-  *btc-node-store
+/+  *btc-node-json, default-agent, verb
+::
+=>  |%
+    ::
+    +$  card  card:agent:gall
+    ::
+    +$  state
+      $%  [%0 state-zero]
+      ==
+    ::
+    +$  state-zero
+      $:  =wallets
+          default-wallet=@t
+      ==
+    --
+::
+=|  state-zero
+=*  state  -
+::  Main
+::
+%+  verb  |
+^-  agent:gall
+=<  |_  =bowl:gall
+    +*  this      .
+        btc-core  +>
+        bc        ~(. btc-core bowl)
+        def       ~(. (default-agent this %|) bowl)
+    ::
+    ++  on-init
+      ^-  (quip card _this)
+      :-  ~
+      %=    this
```

Comment:
Generally use `%_` when you can, it prevents you from accidentally changing the types of things.